gpbft/gpbft.go

```diff
@@ -779,6 +795,7 @@ func (i *instance) getRound(r uint64) *roundState {
 func (i *instance) beginNextRound() {
 	i.log("moving to round %d with %s", i.round+1, i.proposal.String())
 	i.round += 1
+	i.progress.NotifyProgress(i.instanceID, i.round, i.phase)
```

Comment:
I don't think we've updated the phase yet. We might need to move this to `beginConverge`.